ideascube/serveradmin/catalog.py

```diff
@@ -176,6 +185,97 @@ def unregister_site(self, package):
         except FileNotFoundError as e:
             printerr(e)
 
+class MediaCenter(Handler):
+    def commit(self):
+        for pkg in self._removed:
+            self.remove_pkg(pkg)
```

Comment:
Handler already have an `install` method that is supposed to install the package, why not using that?